### ISSUE TITLE
The "new series" dialog in the unrecognised files utility is confusing at first.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dizzydezz/jmm#212]
```
A link to a manual would be nice :).
```

Original issue reported on code.google.com by `Craige1Emmott` on 22 May 2012 at 3:15

**Comment by GoogleCodeExporter** [comment]
```
Maybe adding a link at the top to of the New Series dialogue to:
http://code.google.com/p/jmm/wiki/User_Manual_FileManager#Unrecognized_Files
and expanding the explanation of how to go about it a bit, may be helpful.
```

Original comment by `inferk...@gmail.com` on 22 Nov 2012 at 5:54

**Comment by GoogleCodeExporter** [comment]
```
I had a thought on this. Perhaps moving the group section down and the AniDB 
section up.
By the AniDB section, to:
- remove the radio buttons,
- instead add a search button (between the search box and red "X" button), that 
when pressed, will search AniDB for the anime typed out in the search box, 
showing the list below, as is currently displayed when the "existing" radio 
button is selected, but to have the ones that are not part of the personal 
collection (are only on AniDB), highlighted red or something.
- if any of the options in the list is selected (single-clicked on), then to 
have the main cover art/and or anime overview and/or total number of episodes 
in the series downloaded and displayed in on the right of the same dialogue 
window (to allow one to identify the anime within the dialogue window, without 
even needing to go to AniDB.
- if the "confirm" button (at the bottom) is then pressed while selected option 
is selected, then JMM can automatically add the AniDB ID (which it would have 
gotten when it download the info in the above point) to the collection.

By the Group section, to:
- add an automatic radio button (which will be selected by default), that when 
selected, will automatically add the anime in the "AniDB section" to an 
appropriate existing or new group and sorting list.

Seems like a good idea. As always, it's kind of difficult to explain without 
illustrations, but I hope it makes sense.
```

Original comment by `inferk...@gmail.com` on 6 Dec 2012 at 4:44

**Comment by GoogleCodeExporter** [comment]
Original comment by `werndly...@gmail.com` on 21 Dec 2012 at 2:26
- Added labels: **Priority-High**, **Product-JMM**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 3.1.28

Note - I can't do some of the things you suggested, like download anime 
information on the fly, as that would be too risky in terms of getting a ban.
```

Original comment by `werndly...@gmail.com` on 3 Jan 2013 at 11:17
- Changed state: **Fixed**
